.gitlab-ci.yml

```diff
@@ -28,7 +28,7 @@ variables:
   CARGO_TARGET_DIR:                "/ci-cache/${CI_PROJECT_NAME}/targets/${CI_COMMIT_REF_NAME}/${CI_JOB_NAME}"
   # CI_IMAGE is changed to "-:staging" when the CI image gets rebuilt
   # read more https://github.com/paritytech/scripts/pull/244
-  CI_IMAGE:                        "paritytech/ink-ci-linux:production"
+  CI_IMAGE:                        "paritytech/ink-ci-linux:staging"
```

Comment:
If the CI passes with this we can revert it and merge

Reply:
Okay so the CI didn't pass, and I suspect it has to do with #1547.

I'm gonna say we merge this anyways since we'll need this fix to build the `production` image